addons/account/controllers/portal.py

```diff
@@ -130,3 +143,17 @@ def details_form_validate(self, data, validate_main_address=True):
                 error['company_name'] = 'error'
                 error_message.append(_('Changing your company name is not allowed once invoices have been issued for your account. Please contact us directly for this operation.'))
         return error, error_message
+
+    def _invoice_get_searchbar_inputs(self):
+        values = {
+            'name': {'input': 'name', 'label': _('Search in #'), 'order': 1},
+        }
+        return dict(sorted(values.items(), key=lambda item: item[1]["order"]))
```

Comment:
Isn't it a bit overkill for a single-item dict ?

Seems done everywhere, could be simplified in all addons isn't it ?